app/prisma/migrations/20240208002748_remove_num_logged_calls/migration.sql

```diff
@@ -0,0 +1,11 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `numLoggedCalls` on the `Project` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "Project" DROP COLUMN "numLoggedCalls";
```

Comment:
I think prisma does migrations in a transaction but let's move this ALTER TABLE line to the end of the migration instead of the beginning. If it's not done in a transaction putting this at the beginning will throw errors on logged call creation until next calls finish.